src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12606,7 +12607,7 @@ void Unit::_ExitVehicle(Position const* exitPosition)
         init.SetFall();
 
     init.MoveTo(pos.GetPositionX(), pos.GetPositionY(), height, false);
-    init.SetFacing(GetOrientation());
+    init.SetFacing(pos.GetOrientation());
```

Comment:
should this take into account the current unit orientation like it did before ? at least when scripts don't change it ? especially in case no exit position is being passed as parameter

Reply:
You're right, I meant to have pos contain the current orientation when I changed that, but forgot to actually do it. Thanks!